geoindicators/src/main/groovy/org/orbisgis/DataUtils.groovy

```diff
@@ -14,52 +14,49 @@ import org.orbisgis.processmanagerapi.IProcess
  *
  * @return
  */
-static IProcess joinTables(){
-    processFactory.create("Utility process to join tables in one", [inputTableNamesWithId: Map
-                                , outputTableName: String, datasource: JdbcDataSource], [outputTableName: String],
+static IProcess joinTables() {
+    processFactory.create("Utility process to join tables in one",
+            [inputTableNamesWithId: Map, outputTableName: String, datasource: JdbcDataSource],
+            [outputTableName: String],
             { inputTableNamesWithId, outputTableName, JdbcDataSource datasource ->
 
                 logger.info("Executing Utility process to join tables in one")
 
-                String columnKey
-                String a = "a"
+                def columnKey
```

Comment:
If yes. All constants should be extracted.
Not sure it adds anything to the readability of the code